requirements_builder/__init__.py

```diff
@@ -6,6 +6,7 @@
 # Requirements-Builder is free software; you can redistribute it and/or
 # modify it under the terms of the Revised BSD License; see LICENSE
 # file for more details.
+#
```

Comment:
@greut is there a way to keep the empty line between license and module docstring?